crates/router/src/utils.rs

```diff
@@ -123,13 +123,34 @@ impl<E> ConnectorResponseExt
     }
 }
 
+pub fn is_zero_decimal_currency(currency: diesel_models::enums::Currency) -> bool {
+    matches!(
+        currency,
+        diesel_models::enums::Currency::JPY | diesel_models::enums::Currency::KRW
+    )
+}
+
+pub fn is_three_decimal_currency(currency: diesel_models::enums::Currency) -> bool {
```

Comment:
Let's move this method to Currency implementation. There let's have branch for all the cases, So that for new currency addition people will take conscious decision.

Reply:
Will be picked in separate pr